api/services/user_contact_service.py

```diff
@@ -25,16 +25,32 @@ def init_data(self, jsonData):
     def handle_contact_group(self, jsonData):
         self.init_data(jsonData)
         contact_groups = jsonData["contact"]["groups"]
-        self.mark_user_groups_as_inactive()
+        active_user_groups = models.UserGroup.query.get_by_user_phone(self.user_phone)
```

Comment:
From the variable name, it seems we are fetching only active user groups, but the function definition says fetch all the groups a user has. Do you think we need to update either the variable name or the function definition?